examples/battery_pool.py

```diff
@@ -22,6 +22,9 @@ async def main() -> None:
     logging.basicConfig(
         level=logging.DEBUG, format="%(asctime)s %(name)s %(levelname)s:%(message)s"
     )
+    # Reduce undesired noise from hpack
+    logging.getLogger("hpack.hpack").setLevel(logging.INFO)
```

Comment:
could maybe be configurable through the client?

Reply:
Mmm, though one. I really don't think we should mess with silencing logging levels or loggers in libraries, IMHO that should be up to the end user.

Reply:
Also, please note we are enabling `DEBUG` logging in the above line, I'm actually surprised we are not getting much more crap from that setting. Normally you would have to do a much more conscious configuring of the logging system. I think the SDK eventually could provide some sensible default logging config (and probably a way to update the logging config dynamically, via `ConfigManager`), and I would be fine to set `hpack` logging to `INFO` there by default.